### ISSUE TITLE
Train using Colmap

### ISSUE THREAD

**Issue by yyashpatel** [kwea123/ngp_pl#87]
Hi !

When we train the data using Colmap, it is seen that the frame convention [right, down, forward] (that colmap uses) is usually converted to [right, up, back]. But as I went through your code I see that you did not perform this operation, could you please comment on this ? 
It would be great if you could help me with it .

Thanks !

**Comment by kwea123** [comment]
Actually any axes direction is fine so long as the `get_rays` function follows the same direction.
For eample, in my `get_rays` function, the rays are generated like:
https://github.com/kwea123/ngp_pl/blob/8d04855eb3c2a6924a5f2a9a8eaa3720758fec51/datasets/ray_utils.py#L29-L32
which follows [right, down, forward] direction.

The original NeRF uses [right, up, back] and their `get_rays` function is https://github.com/bmild/nerf/blob/master/run_nerf_helpers.py#L127 (notice the minus sign for y and z components)

So basically you can use whatever axes direction, I use [right, down, forward] directly following COLMAP convention
